src/40_Dynamic_AMP/Dynamic_Content.html

```diff
@@ -0,0 +1,45 @@
+<!-- ## Dynamic Content -->
+<!--
+This is a sample showing how to show dynamic content based on user interaction: here we are showing
```

Comment:
s/based on/after an

Reply:
s/here we are showing how to show the product availability based on the product selection/the product availability will be shown based on product selection.

Reply:
done